unity-renderer/Assets/UIComponents/Scripts/Components/GridContainer/GridContainerComponentView.cs

```diff
@@ -137,11 +157,11 @@ public BaseComponentView GetItem(int index)
 
     public List<BaseComponentView> GetAllItems() { return instantiatedItems; }
 
-    internal void CreateItem(BaseComponentView newItem, string name)
+    internal void CreateItem(BaseComponentView newItem, string name, bool instantiateNewCopyOfItem = true)
     {
         if (Application.isPlaying)
```

Comment:
Let's use preprocessor directives for this

Reply:
Same as above.